core/services/relay/evm/write_target.go

```diff
@@ -30,6 +34,47 @@ import (
 	relayevmtypes "github.com/smartcontractkit/chainlink/v2/core/services/relay/evm/types"
 )
 
+// EVM Data-Feeds specific processor decodes writes as 'data-feeds.registry.FeedUpdated' messages + metrics
+type dataFeedsProcessor struct {
```

Comment:
No reason this should be here vs. `chainlink-evm`

Reply:
moved to chainlink-evm